python/pyspark/pandas/frame.py

```diff
@@ -8922,7 +8920,7 @@ def update(self, other: "DataFrame", join: str = "left", overwrite: bool = True)
         1  b  e
         2  c  f
 
-        For Series, it's name attribute must be set.
+        For Series, its name attribute must be set.
```

Comment:
For Series, its name attribute must be set.
its or it's ?

Reply:
Use "its"
Confusingly, that is the possessive, with no apostrophe, even though we'd say "Bjorn's" to mean your possession
"it's" = "it is"